### ISSUE TITLE
Support for arm64

### ISSUE THREAD

**Issue by nottooshort** [cyralinc/terraform-provider-cyral#229]
With m1 macs becoming more common, can you add support for arm64?

Terraform version 1.1.2
cyralinc/cyral v2.3.1
```
│ Error: Incompatible provider version
│
│ Provider [registry.terraform.io/cyralinc/cyral](http://registry.terraform.io/cyralinc/cyral) v2.3.1 does not have a package available for your current platform, darwin_arm64.
│
│ Provider releases are separate from Terraform CLI releases, so not all providers are available for all platforms. Other versions of
│ this provider may have different platforms supported.
```

We've otherwise been forced to hard lock our homebrew and tfenv to amd64. This is a hack we must apply to utilize the Cryal provider

```
arch -x86_64 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
arch --x86_64 brew reinstall tfenv
export TFENV_ARCH=amd64

**Comment by wcmjunior** [comment]
Hey @nottooshort , thanks for your contribution.

`darwin_arm64` is already supported since `v2.5.1` (see #160). If you are not using Pager Duty integration you can upgrade to the latest version of the provider.

Please reach out to the customer success team if you need help with the upgrade.

Let me know if that helps so I can close the ticket.

**Comment by wcmjunior** [comment]
Closed as `darwin_arm64` is available since `v2.5.1` and the upgrade can be performed if Pager Duty integration is not used.
